### ISSUE TITLE
Cannot start nuxt:  Missing "./esm/formatInTimeZone" specifier in "date-fns-tz" package   

### ISSUE THREAD

**Issue by teknotut** [becem-gharbi/nuxt-naiveui#26]
I'm not sure if this is an issue on my end or not, but prior to the 'npm update,' @bg-dev/nuxt-naiveui was working fine. However, after running `npm update`, I encountered an error like the one below:

```
ERROR  Cannot start nuxt:  Missing "./esm/formatInTimeZone" specifier in "date-fns-tz" package                           3:39:54 PM

  at e (node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:21455:25)
  at n (node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:21455:627)
  at o (node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:21455:1297)
  at resolveExportsOrImports (node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:28751:20)
  at resolveDeepImport (node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:28770:31)
  at tryNodeResolve (node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:28458:20)
  at Context.resolveId (node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:28217:28)
  at Object.resolveId (node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:44279:64)
  at async node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:65919:21
  at async node_modules/@nuxt/vite-builder/node_modules/vite/dist/node/chunks/dep-3bba9c7e.js:44924:20
```

Has anyone else experienced this error?"

Here my package.json
```
{
  "name": "nuxt-app",
  "private": true,
  "type": "module",
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "@bg-dev/nuxt-naiveui": "^1.4.7",
    "@nuxt/types": "^2.17.1",
    "@nuxtjs/tailwindcss": "^6.8.0",
    "nuxt": "^3.7.4",
    "nuxt-icon": "^0.5.0"
  },
  "dependencies": {
    "@pinia/nuxt": "^0.4.11",
    "@tailwindcss/forms": "^0.5.6",
    "pinia": "^2.1.6",
    "preline": "^1.9.0"
  },
  "overrides": {
    "vue": "latest"
  }
}
```

**Comment by teknotut** [comment]
The problem only in dev mode, but work in production mode.

**Comment by becem-gharbi** [comment]
Thanks for reporting this issue. It should be solved on [v1.5.1](https://github.com/becem-gharbi/nuxt-naiveui/releases/tag/v1.5.1).
